vagrant_provision_script.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+sudo apt-get update
+
+if grep "#Your Custom Settings" ~/.bashrc 
```

Comment:
Nice use of these checks to allow almost-idempotent provisioning!

Reply:
My eyes are not getting any younger ... put this in so I could actually read the bash shell output :-) Otherwise in windows when you ssh in to play around its very hard to read.